lib/apiservers/engine/backends/backends.go

```diff
@@ -97,6 +97,15 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 		log.Errorf("Failed to create image store")
 		return err
 	}
+	log.Info("Refreshing repo cache")
+	go func() {
+		// TODO: this might move to the image cache refresh
+		if err := cache.NewRepositoryCache(portLayerClient); err != nil {
```

Comment:
`cache.rCache` is a pointer which gets populated in `NewRepositoryCache`.  Since this is run in an anonymous func, we may not assign `rCache` by the time it's needed.  Won't that result in a panic?